### PR TITLE
[ci skip] Clarify Hyperlinks Example for Internal Links

### LINKED ISSUE #160: How to use custom text for a hyperlink
If we wanted to use custom text for a hyperlink how would we do that.

I see the [hyperlinks example/docs](https://github.com/caxlsx/caxlsx/blob/master/examples/hyperlink_example.md) shows some sort of custom text for the "Internal" link to the "Next Sheet" however its not obvious to me if you can do custom text for "External" links

I would be happy to help update the docs to clarify if someone can chime in here.

### PR DESCRIPTION
**Pull request by westonganger**
### Description
Closes #160 

### Your checklist for this pull request

- [x] I have reviewed the [guidelines for contributing](../blob/master/CONTRIBUTING.md) to this repository.
- [ ] I have added (or updated) appropriate tests if this PR fixes a bug or adds a feature.
- [x] If this PR doesn't need tests (docs change), I added `[ci skip]` to the title of the PR.
- [x] If this closes any issues, I have added "Closes `#issue`" to the PR description or my commit messages.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ ] I added an entry to the [changelog](../blob/master/CHANGELOG.md).

### CODE REVIEW

File: examples/hyperlink_example.md [hunk 1]
```diff
@@ -10,17 +10,22 @@ require 'axlsx'
...
+  
+  # Internal References (Links pointing to within the Workbook)
   sheet.add_row ['next sheet']
-end
-
-wb.add_worksheet(name: 'Next Sheet') do |sheet|
-  sheet.add_row ['hello!']
+  sheet.add_hyperlink(
+    location: "'Next Sheet'!A1", ### This link will point to Cell A1 of the Worksheet named "Next Sheet".
+    target: :sheet, ### This is :external by default. If you set it to anything else, the location is interpreted to be the current workbook.
```
Comment (westonganger) [inline examples/hyperlink_example.md:26]: This comment text for `:target` was copied from https://github.com/caxlsx/caxlsx/blob/946f287a37c7a5d2b4fe2ae8d109848e257241b1/lib/axlsx/workbook/worksheet/worksheet_hyperlink.rb#L16

**Comment by noniq** [comment]
Wow, thanks for tackling this! Would you mind update the screenshot (https://github.com/caxlsx/caxlsx/blob/master/examples/images/hyperlink_example.png) to match the updated code?

**Comment by westonganger** [comment]
@noniq ok updated with a screenshot (and some more improvements to make screenshot/example more readable)

**Comment by noniq** [comment]
Excellent work! Thanks a lot!

**Pull request merged**